docs/controller.md

```diff
@@ -0,0 +1,46 @@
+
+## Objective
+The Controller component plays a central role in the CI/CD system by orchestrating the flow of actions from the actions received, breaking these down into specific action steps, and managing communication between the Monitor, the Scheduler and the Agents.
+
+### Main features
+1. Receiving CI triggers:
+
+    a. The Controller receives CI triggers from the Monitor component.
```

Comment:
In the other documentation file, you say this is done by the "user" and not the "Monitor component"?